src/components/EvidenceTile.tsx

```diff
@@ -21,12 +23,19 @@ export const EvidenceTile: FunctionComponent<Props> = (props) => (
         {props.purpose && <> with {props.purpose}</>}
       </p>
     </div>
+
     {props.toReview && (
       <div className={`lbh-body-s ${styles.actions}`}>
-        <Link href="/accept">
+        <Link
+          href={`/dashboard/resident/${props.residentId}?action=accept`}
```

Comment:
How does it know which document it's accepting if it's linking to the resident page? Should think link to the document page itself with the accept action?

Reply:
this is what i think we'll need to redesign. i've been thinking about ways to do the routes, like this: https://github.com/vercel/next.js/tree/canary/examples/with-route-as-modal

Reply:
That would be a nice iteration, definitely 👍 but in the meantime, does it mean we can delete some code from here? 

Even with the modal route, I'm not sure I understand how this will work without the document ID to know which document to accept/reject, should the dialogs just live on just the documents page and this can link to there?

Reply:
we can do that for now, yes. i was thinking about structuring routes like: `?accept=123` and `?reject=456` and so on

Reply:
That could work, I'm not sure why we'd need multiple routes to accept/reject when we already have a URL which includes the document submission ID.

I'm actually wondering why we need a route for these actions at all—typically GET requests should be non-mutating. Could we not just make the API request to accept/reject when they hit the accept/reject button, and leave them where they were?

Reply:
the actions need explanation, especially for novice users (and rejecting needs a reason to be given)

Reply:
Not sure we're talking about the same thing 😅 We can still show the modal, but it can be shown in place without a URL change

Reply:
seemed like less code than managing cancel handlers and state hooks inside the app

Reply:
also felt like it would be more scalable if they get shown on different screens (like the resident detail screen, which we took away for now). i tend to expose state in the url unless there's a good reason not to

Reply:
the code would be the same, just called from the modal instead of the route.

I wasn't suggesting taking away the resident detail screen. Just that take the modals off it and link to the document screen for accept/reject.